auto-transcoder.sh

```diff
@@ -7,25 +7,22 @@ URL="";
 FILENAME="";
 
 transcodeVideo() {
-	ID=`youtube-dl "$URL" --get-id`;
 
 	if [[ $? == 0 ]]; then
-		TEMP="./$ID.webm";
-
 		TARGET="$FILENAME.ogg";
 
-		youtube-dl "$URL" -f 171 --id -q;
+		youtube-dl "$URL" -f bestaudio/best -q -o "audiostream";
```

Comment:
I preferred `-o` over `-x` because if the format downloaded is bestaudio, it's already an audio file and if the format downloaded is best, the whole video+audio file is downloaded and then audio is extracted, which defeats the purpose of the utility as it does two conversions instead of one.

Reply:
This will work of all future formats by the way.